From e690755f23dd4ac17df7160a20819a3194cbdd34 Mon Sep 17 00:00:00 2001
Message-Id: <e690755f23dd4ac17df7160a20819a3194cbdd34.1560237392.git.maciej.zenon.borzecki@canonical.com>
From: Maciej Borzecki <maciej.zenon.borzecki@canonical.com>
Date: Mon, 10 Jun 2019 13:37:44 +0200
Subject: [PATCH] data/selinux: bump SELinux policy module version

Signed-off-by: Maciej Borzecki <maciej.zenon.borzecki@canonical.com>
---
 data/selinux/snappy.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/selinux/snappy.te b/data/selinux/snappy.te
index a63b88e0a42370e323704590bc0966d969ec6971..8b7834e048802f801d13ca213e0344803876488c 100644
--- a/data/selinux/snappy.te
+++ b/data/selinux/snappy.te
@@ -17,7 +17,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 
-policy_module(snappy,0.0.14)
+policy_module(snappy,0.1.0)
 
 ########################################
 #
-- 
2.22.0

