From ebc1b5c3a1adc6d7e5287fa76811968e7faa7621 Mon Sep 17 00:00:00 2001
From: Simon Fels <simon.fels@canonical.com>
Date: Thu, 30 Mar 2017 17:39:44 +0200
Subject: [PATCH] cmd/snap-confine: hardcode constants from xfs/xqm.h for now

---
 cmd/snap-confine/seccomp-support.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/cmd/snap-confine/seccomp-support.c b/cmd/snap-confine/seccomp-support.c
index a08b23a..18e0459 100644
--- a/cmd/snap-confine/seccomp-support.c
+++ b/cmd/snap-confine/seccomp-support.c
@@ -34,7 +34,18 @@
 #include <sys/types.h>
 #include <sys/utsname.h>
 #include <termios.h>
+#if 0
 #include <xfs/xqm.h>
+#else
+#define XQM_CMD(x)	(('X'<<8)+(x))	/* note: forms first QCMD argument */
+#define Q_XQUOTAON	XQM_CMD(1)	/* enable accounting/enforcement */
+#define Q_XQUOTAOFF	XQM_CMD(2)	/* disable accounting/enforcement */
+#define Q_XGETQUOTA	XQM_CMD(3)	/* get disk limits and usage */
+#define Q_XSETQLIM	XQM_CMD(4)	/* set disk limits */
+#define Q_XGETQSTAT	XQM_CMD(5)	/* get quota subsystem status */
+#define Q_XQUOTARM	XQM_CMD(6)	/* free disk space used by dquots */
+#define Q_XQUOTASYNC	XQM_CMD(7)	/* delalloc flush, updates dquots */
+#endif
 #include <unistd.h>
 
 #include <seccomp.h>
-- 
2.7.4