diff --git a/cmd/Makefile.am b/cmd/Makefile.am
index 8bb371e..ee5a79e 100644
--- a/cmd/Makefile.am
+++ b/cmd/Makefile.am
@@ -10,10 +10,6 @@ noinst_LIBRARIES =
 CHECK_CFLAGS = -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes \
 	-Wno-missing-field-initializers -Wno-unused-parameter
 
-# Make all warnings errors when building for unit tests
-if WITH_UNIT_TESTS
-CHECK_CFLAGS += -Werror
-endif
 
 subdirs = snap-confine snap-discard-ns system-shutdown libsnap-confine-private
 
